package.json

```diff
@@ -10,13 +10,15 @@
     "body-parser": "~1.12.0",
     "bower": "^1.7.9",
     "cookie-parser": "~1.3.4",
+    "crypto": "0.0.3",
     "debug": "~2.1.1",
     "ejs": "~2.3.1",
     "express": "~4.12.2",
     "mongoose": "^4.6.0",
     "morgan": "~1.5.1",
     "mssql": "^3.3.0",
     "passport-localapikey": "0.0.3",
-    "serve-favicon": "~2.2.0"
+    "serve-favicon": "~2.2.0",
+    "sha1": "^1.1.1"
```

Comment:
This should be a part of the frontend application.  Hashing done on the server means that the password is sent over the wire in cleartext which is a security risk.